### ISSUE TITLE
Looking for the configurations and parameter for this experiment

### ISSUE THREAD

**Issue by kaihuchen** [lipiji/cws-seq2seq#2]
Wonder if you would be so kind as to share the configurations and parameters that you used to run this experiment. Thanks!

**Comment by lipiji** [comment]
Hi,

I just use a typical attention based seq2seq framework. 
hidden state size=500.
embedding size for word = 300
beam size for beam search = 10
adadelta as learning method.

**Comment by kaihuchen** [comment]
Somehow I am having trouble duplicating your result using the same dataset. Guess I must be doing something wrong. 

I trained the model  at character level (i.e., with delimiter set to empty string), target dataset being the  Training took a while but seems to converge just fine. But when I tried to make predictions (using infer.py), it reports an UnicodeDecodeError  (documented here https://github.com/google/seq2seq/issues/299 ). I traced the problem to what looks like a bug in seq2seq.tasks.decode_text.py .   Since you did not run into the same problem, I figure you must be doing something different.  Are you using the standard [Google seq2seq](https://github.com/google/seq2seq) for this?
